assets/javascript/game.js

```diff
@@ -4,42 +4,61 @@ var guessWrong = 0;
 var maxGuesses = 7;
 var winCount = 0;
 var lossCount = 0;
-var wordBank = ["Algebra", "Poodle", "Citizen", "Pizzaz", "Cooperate", "Munchkin", "Flower", "Supercalifragilisticexpialidocious"];
+var wordBank = ['Algebra', 'Poodle', 'Citizen', 'Pizzaz', 'Cooperate', 'Munchkin', 'Flower', 'Supercalifragilisticexpialidocious'];
 
-//Pull from wordBank to give puzzle word
+// Pull from wordBank to give puzzle word, and make sure all buttons are enabled
+// Reset game state to initial state
+// Can add an event listener to a reset button to use this method to reset the game.
 function initializeGame() {
     var gameWord = wordBank[Math.floor(Math.random() * wordBank.length)];
     var answerContainer = document.getElementById('answer');
-    answerContainer.innerHTML = "";
+    answerContainer.innerHTML = '';
+
     for (var i = 0; i < gameWord.length; i++) {
         var letter = gameWord[i];
-        var upperLetter = letter.toUpperCase(); 
-        answerLetters[upperLetter] = true;  //making sure all cases of letter work
-        var letterDiv = document.createElement('div');
-        letterDiv.classList.add(['letter-container'])
-        letterDiv.classList.add(['p-3'])
-        var letterSpan = document.createElement('span');
-        letterSpan.setAttribute('name', upperLetter);
-        letterSpan.innerHTML = letter;
-        letterSpan.classList.add(['hide-letter']);  //using class to hide display of letters
-        var underscoreSpan = document.createElement('span');
-        underscoreSpan.innerHTML = '_'; //creating "_" to display under letters
-        underscoreSpan.classList.add(['under-letter'])
-        letterDiv.appendChild(letterSpan);
-        letterDiv.appendChild(underscoreSpan);
-        answerContainer.appendChild(letterDiv);
+        var upperLetter = letter.toUpperCase();
+        answerLetters[upperLetter] = true; //making sure all cases of letter work
+        answerContainer.appendChild(createAnswerDiv(gameWord, letter, upperLetter));
     }
+
+    enableButtons();
+    // TODO Hide win/loss banner
 }
 
-initializeGame();
+function enableButtons() {
+    // TODO Go through all buttons and re-enable them.
+}
 
+// I moved this logic into a function and call it on line 30 instead of having this all inline in the for loop.
+// This makes it easier to read
+function createAnswerDiv(gameWord, letter, upperLetter) {
+    var letterDiv = document.createElement('div');
+    letterDiv.classList.add(['letter-container']);
+    letterDiv.classList.add(['p-3']);
 
-// Variables to create multiple buttons on screen
-var letters = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"];
```

Comment:
I moved these variables withing the `createButtons` function as that was the only place they were used.